module_2/gifts/service/src/main/java/com/epam/esm/service/validation/Util.java

```diff
@@ -0,0 +1,21 @@
+package com.epam.esm.service.validation;
+
+import org.springframework.util.StringUtils;
+
+public class Util {
+	public static String removeExtraSpaces(String source) {
+		if(source != null) {
+			return source.replaceAll("\\s+", " ").trim();
+		}
+		return null;
```

Comment:
@LizavetaSinitsyna Try to avoid return `null`

Reply:
Could we leave here the null as return value taking into account that the logic of the method is to remove all spaces and as null does't contain spaces the method simply return the passed parameter without any changes?